packages/plugin-default-event-tracking-advanced-browser/src/libs/messenger.ts

```diff
@@ -91,14 +105,21 @@ export class WindowMessenger implements Messenger {
       if (action === 'ping') {
         this.notify({ action: 'pong' });
       } else if (action === 'initialize-visual-tagging-selector') {
+        const actionData = eventData?.data as InitializeVisualTaggingSelectorData;
         asyncLoadScript(AMPLITUDE_VISUAL_TAGGING_SELECTOR_SCRIPT_URL)
           .then(() => {
             // eslint-disable-next-line
             amplitudeVisualTaggingSelectorInstance = (window as any)?.amplitudeVisualTaggingSelector?.({
-              visualHighlightClass: AMPLITUDE_VISUAL_TAGGING_HIGHLIGHT_CLASS,
               getEventTagProps,
+              isElementSelectable: (element: Element) => {
+                if (isElementSelectable) {
+                  return isElementSelectable(actionData?.actionType || 'click', element);
```

Comment:
use ActionType.click instead?

Reply:
The string literal union type here doesn't work like this, intended to use string literal type instead of enum as it is easier to deal with.